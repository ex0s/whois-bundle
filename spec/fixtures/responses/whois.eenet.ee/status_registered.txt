
The registry database contains only .EE, .COM.EE, .PRI.EE, .FIE.EE,
.ORG.EE and .MED.EE domains.

Registrant:
   ADVOKAADIBÜROO SORAINEN AS
   PÄRNU MNT, 15, HARJUMAA TALLINN KESKLINN 10141
   TEL 5274536
   FAX +372 6400 901

   Domain Name: google.ee

   Contacts:
      Mart Meier      mart.meier@sorainen.ee
      Joshua Hopping      ccops@markmonitor.com

   Record created on 22-Apr-2003
   Record changed on 28-May-2010

   Domain servers:
      ns1.google.com.
      ns2.google.com.

Registrar: EENET
URL: http://www.eenet.ee/EENet/dom_reeglid.html (in estonian)
URL: http://www.eenet.ee/EENet/ee-tldomreg.html

