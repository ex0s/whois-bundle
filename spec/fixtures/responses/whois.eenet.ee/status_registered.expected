#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2003-04-22}

#updated_on
  %s %CLASS{time}
  %s %TIME{2010-05-28}

#expires_on
  %s %ERROR{AttributeNotSupported}


#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.google.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.google.com"
